File: LAPORAN PRATIKUM/MODUL4/CODEBLOKS DAN PYTHON/SOAL 4/PRAK304-2210817210014-MuhammadQoharyRawidafany.py
I = int (input(""))
if I >= 1 and I < 10 :
    print("Satuan")
elif I >= 10 and I < 19 :
    print("Belasan")
elif I >= 20 and I < 99 :
    print("Puluhan")
elif I == 0 :
    print("Nol")
else :
    print("Anda Menginput Melebihi Limit Bilangan")

File: LAPORAN PRATIKUM/MODUL4/CODEBLOKS DAN PYTHON/SOAL 4/PRAK304-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
int main()
{
    for(int q=1;q<=5;q++){
    int I;
    printf("");
    scanf("%d",&I);
    if(I>=1 && I<=9){
    printf("Satuan\n");
    }
    else if (I>=10 && I<=19){
        printf("Belasan\n");
    }
    else if (I>=20 && I<=99){
        printf("Puluhan\n");
    }
    else if (I==0){
        printf("Nol\n");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }
    }
}
